hdfscm/checkpoints.py

```diff
@@ -1,12 +1,16 @@
 import hashlib
+import os
 import posixpath
-from notebook.services.contents.checkpoints import Checkpoints
+
+if os.getenv('JUPYTER_ENV') == 'dev':
+    from notebook.services.contents.checkpoints import Checkpoints
+else:
+    from jupyter_server.services.contents.checkpoints import Checkpoints
+from pyarrow import fs
```

Comment:
The test methods still require the notebook dependencies (they don't exist in `jupyter_server` package).